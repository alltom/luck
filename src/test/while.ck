while(false)
    <<< "don't print" >>>;

while(int i < 2) {
    <<< "print twice" >>>;
    i + 1 => i;
}

<<< "done" >>>;
