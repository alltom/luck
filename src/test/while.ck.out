print twice
print twice
done
